tests/js/lucene-tests.js

```diff
@@ -1,5 +1,9 @@
 // Sanity checks for search integration.  As these are read only, we use a single instance for all tests.
 "use strict";
+
+var events = require("events");
+events.EventEmitter.defaultMaxListeners = 100;
```

Comment:
The tests in this package no longer trigger the warning.  I have removed these lines for now.  If the problem crops up when running larger sets of tests in other packages (such as the UL API), we can revisit this there.